src/core/js/views/buttonsView.js

```diff
@@ -96,28 +92,29 @@ define(function() {
                 var buttonText = this.model.get('_buttons')["_" + changedAttribute].buttonText;
 
                 switch (changedAttribute) {
-                    case "showCorrectAnswer": case "hideCorrectAnswer":
+                    case "submit":
+                    case "showCorrectAnswer":
+                    case "hideCorrectAnswer":
     				    //make model answer button inaccessible but enabled for visual users
     				    //	due to inability to represent selected incorrect/correct answers to a screen reader, may need revisiting
-                        this.$('.buttons-action').a11y_cntrl(false).html(buttonText).attr('aria-label', ariaLabel);
+                        this.$('.buttons-action').a11y_cntrl_enabled(false).html(buttonText).attr('aria-label', ariaLabel);
```

Comment:
I've reverted the function back and I'm still able to see the issue I mentioned.

I have a simple assessment with two MCQ components with 2 allowed attempts and suppress marking enabled. On submission of the second attempt and completion of the assessment (having answered incorrectly on both questions), the 'Show correct answer' button is disabled on the first MCQ, second is enabled fine.

It seemed to be down to the fact that the submit button is initially disabled, then when 'a11y_cntrl_enabled(false)' occurs, it disables the aria stuff, but doesn't explicitly enable the button. Hopefully that makes sense.

I've resolved the issue in this commit by ensuring the button is enabled first:
https://github.com/adaptlearning/adapt_framework/pull/1360/commits/7f0e4ab43fa5906ddd8ce7ee0ae652e72bf22fb9